packages/toolkit/src/extension.ts

```diff
@@ -174,7 +130,7 @@ export async function activate(context: vscode.ExtensionContext) {
             context: extContext,
             regionProvider: globals.regionProvider,
             toolkitOutputChannel: globals.outputChannel,
-            remoteInvokeOutputChannel,
+            remoteInvokeOutputChannel: globals.invokeOutputChannel,
```

Comment:
(off-topic) Another thing that has been in the backlog: there isn't really a strong reason to have a separate output channel for "remote invoke", and even if there were a reason, it should be created on-demand, not on startup...